src/vs/workbench/browser/parts/compositeBar.ts

```diff
@@ -706,18 +719,23 @@ class CompositeBarModel {
 				item.visible = true;
 				changed = true;
 			}
+
+			this.updateItemsOrder();
 			return changed;
 		} else {
 			const item = this.createCompositeBarItem(id, name, order, true, true);
 			if (isUndefinedOrNull(order)) {
 				this.items.push(item);
 			} else {
 				let index = 0;
+				console.log('adding new item');
```

Comment:
Forgot to remove `console.log` ?

Reply:
will clean up and merge, thanks for reviewing